### PR TITLE
PEP-8 code standards

### PR DESCRIPTION
**Pull request by isht3**
* Run flake8 on travis

* Flake8 configuration - setup.cfg

### CODE REVIEW

**Comment by farizrahman4u** [comment]
Please get the build to pass. Thanks.

File: eywa/math/numpy_backend.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
+# flake8: noqa
```
Comment (isht3) [inline eywa/math/numpy_backend.py]: @farizrahman4u This one line in the file breaks the tests for numba. The rest all works properly. Tested. :+1: Should I remove this? It runs properly then.

Comment (farizrahman4u) [reply eywa/math/numpy_backend.py]: I dont get it, what has backend got anything to do with this?
They way it should work is, there should be 3 jobs: one for numba backend, one for numpy and one for pep8.

The pep8 job shouldn't run any tests, just check pep8.

Comment (farizrahman4u) [reply eywa/math/numpy_backend.py]: You can check the keras repo for reference.

Comment (isht3) [reply eywa/math/numpy_backend.py]: @farizrahman4u I'm not quite sure how flake8 can be run on it's own without running tests. I'll look into it.

Comment (farizrahman4u) [reply eywa/math/numpy_backend.py]: Any updates? Thanks.

Comment (isht3) [reply eywa/math/numpy_backend.py]: https://github.com/keras-team/keras/blob/adfec1e8440b3af64b642ea75bdb21bbee70cc80/.travis.yml#L10 This the PEP8 tests on keras, right?

Comment (farizrahman4u) [reply eywa/math/numpy_backend.py]: Yes, and here.. https://github.com/keras-team/keras/blob/adfec1e8440b3af64b642ea75bdb21bbee70cc80/.travis.yml#L110

**Pull request closed without merging**